NEWS

```diff
@@ -5,6 +5,9 @@ rmarkdown 1.11 (unreleased)
 
 * The icons in `flexdashboard::valueBox()` are not of the full sizes due to the upgrade of FontAwesome in #1340 in the previous version (#1388, rstudio/flexdashboard#189).
 
+* Fix for `run()` to work if `file = NULL` and multiple .Rmd files are present in the search
+path (#1406).
```

Comment:
It will be good to mention #906, too.

nit: we have never hard-wrapped lines in NEWS.md.

Reply:
Just changed this. Thanks! Do we cite PR and issue numbers interchangeably?

Reply:
It is not strictly necessary to cite both, but redundancy doesn't hurt here (show both the bug number and PR number).

Reply:
Okay, great, will do. Just noticed a ton of line breaks in NEWS at earlier version numbers. Could I take a pass through it and remove them all? (A consistency play.)

Reply:
Yes, please (if you have time -- apparently this is not very important).